glide.yaml

```diff
@@ -72,3 +71,4 @@ import:
 - package: github.com/Shopify/sarama
   version: ^1.11.0
   repo: http://github.com/Shopify/sarama
+- package: github.com/rcrowley/go-metrics
```

Comment:
The strong reason is that the Kafka client library depends on this to provide metrics, hence we don't get a choice. Either this, or no Kafka metrics.